changes/v4.3.22-en.md

```diff
@@ -45,6 +45,9 @@
 - For Rule-Engine resource creation failure, delay before the first retry [#9313](https://github.com/emqx/emqx/pull/9313).
   Prior to this change, the retry delay was added *after* the retry failure.
 
+- Enhanced security for file access [#9319](https://github.com/emqx/emqx/pull/9319).
```

Comment:
Path traversal attacks are still possible, since "/../" is not sanitized in the input.